scripts/test.js

```diff
@@ -0,0 +1,86 @@
+var assertTrue = require('./assert').assertTrue;
```

Comment:
This file is for testing in the dev stage only and can be removed without any concern :)